### DISCUSSION TITLE
New GITS features to add

### DISCUSSION THREAD

**Discussion by krishnasaurabh** [arjunptm/GITS#58] [Ideas]
This post is to get ideas from the community for the use cases that can be implemented in the gits project.

You can use the below questions to get inspired and start contributing to the repository.

- Are there git commands that you use in your daily and need multiple commands to complete the functionality?
- Do you keep forgetting that one command that does a simple thing and you wanted to create a custom command for it?
- Want to associate different commands (like running a test set before committing) ?

Please comment on your thoughts and if you'd wish to contribute to our project please follow the [CONTRIBUTING.md](https://github.com/arjunptm/GITS/blob/master/CONTRIBUTING.md) for more information.

**Comment by arjunptm** [comment]
Wonderful discussion topic, thank you for bringing this up!  I think this will be a great starting point for new teams to work on.

We should try to add ssh-agent related features to help to manage daily login issues.

**Comment by rhnprabhune** [comment]
Interesting discussion !
